pkg/provider-local/imagevector/images.yaml

```diff
@@ -2,11 +2,11 @@ images:
 - name: machine-controller-manager
```

Comment:
I think we can entirely delete this image since MCM is now managed by `gardenlet` - probably we just forgot about cleaning this up.

Reply:
You are right! Thank you!